delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchViewJsonLdSourceDecoder.scala

```diff
@@ -162,24 +178,39 @@ object ElasticSearchViewJsonLdSourceDecoder {
       )
     case i: IndexingElasticSearchViewFields       =>
       IndexingElasticSearchViewValue(
+        name = i.name,
+        description = i.description,
         resourceTag = i.resourceTag,
-        // If there is no pipeline defined, we use a default one to keep the historic behaviour
+        // If no pipeline defined, we use a default one to keep the historic behaviour
         pipeline = i.pipeline.getOrElse(IndexingElasticSearchViewValue.defaultPipeline),
         mapping = Some(i.mapping),
         settings = i.settings,
         context = i.context,
         permission = i.permission
       )
     case a: AggregateElasticSearchViewFields      =>
-      AggregateElasticSearchViewValue(a.views)
+      AggregateElasticSearchViewValue(
+        name = a.name,
+        description = a.description,
+        views = a.views
+      )
   }
 
-  def apply(uuidF: UUIDF, contextResolution: ResolverContextResolution)(implicit api: JsonLdApi) =
-    new ElasticSearchViewJsonLdSourceDecoder(
-      new JsonLdSourceResolvingDecoder[ElasticSearchViewRejection, ElasticSearchViewFields](
-        contexts.elasticsearch,
-        contextResolution,
-        uuidF
+  def apply(uuidF: UUIDF, contextResolution: ResolverContextResolution)(implicit api: JsonLdApi) = {
+    implicit val rcr = contextResolution.rcr
```

Comment:
Not in the scope of this PR but I think there is something broken (and always was I guess) about `JsonLdSourceResolvingDecoder`
It is far-fetched and nobody uses this feature but with it, users should be able to use custom contexts to create an Elasticsearch view for example.
As nobody uses it, we can discuss about using the regular one that relies only on the default contexts that delta provides (the resolving one just makes things more complex imho)

Reply:
Would we expect to be able to pass `contexts.elasticsearch` and `contextResolution` to the `JsonLdSourceResolvingDecoder` without having to do this for-comprehension and getting this implicit `rcr`?

Reply:
What you did is fine, I just think that in a further PR we can make the whole decoding simpler (that is beyond adding a name and a description)